editor/src/clj/editor/render_pb.clj

```diff
@@ -70,43 +70,43 @@
+                         :label "Render Resources"
                          :columns [{:path [:name]
                                     :label "Name"
                                     :type :string
-                                    :default "New Material"}
-                                   {:path [:material]
-                                    :label "Material"
+                                    :default "New Render Resource"}
+                                   {:path [:path]
+                                    :label "Render Resource"
                                     :type :resource
-                                    :filter "material"
+                                    :filter ["material" "render_target"]
                                     :default nil}]}]}]})
 
 (defn- set-form-op [{:keys [node-id] :as user-data} path value]
   (condp = path
     [:script]          (g/set-property! node-id :script value)
-    [:named-materials] (let [graph-id (g/node-id->graph-id node-id)]
-                         (g/transact
-                           (concat
-                             (for [[named-material-id _] (g/sources-of node-id :named-materials)]
-                               (g/delete-node named-material-id))
-                             (for [{:keys [name material]} value]
-                               (make-named-material-node graph-id node-id name material)))))))
-
-(g/defnk produce-form-data [_node-id script-resource named-materials]
+    [:named-render-resources] (let [graph-id (g/node-id->graph-id node-id)]
+                              (g/transact
+                                (concat
+                                  (for [[named-render-resource-id _] (g/sources-of node-id :named-render-resources)]
+                                    (g/delete-node named-render-resource-id))
+                                  (for [{:keys [name path]} value]
+                                    (make-named-render-resource-node graph-id node-id name path)))))))
+
+(g/defnk produce-form-data [_node-id script-resource named-render-resources]
   (-> form-sections
       (assoc :form-ops {:user-data {:node-id _node-id}
                         :set set-form-op})
       (assoc :values {[:script] script-resource
-                      [:named-materials] named-materials})))
+                      [:named-render-resources] named-render-resources})))
 
-(g/defnk produce-pb-msg [script-resource named-materials]
+(g/defnk produce-pb-msg [script-resource named-render-resources]
   {:script (resource/resource->proj-path script-resource)
-   :materials (mapv (fn [{:keys [name material]}]
+   :render-resources (mapv (fn [{:keys [name path]}]
                       {:name name
-                       :material (resource/resource->proj-path material)})
-                    named-materials)})
+                       :path (resource/resource->proj-path path)})
+                    named-render-resources)})
```

Comment:
Indentation looks messed up here. Select lines, then **Code > Auto-Indent Lines** should fix it!